misc/mallard.service

```diff
@@ -5,11 +5,10 @@ After=network.target
 [Service]
```

Comment:
yes, bless you for moving this into a subdirectory. i was too impressed by his holiness striking's presence in the interdimensional cortex to ask him myself, thank you

Reply:
why do you need docker for a discord bot

Reply:
what steps are there for deploying a discord bot, and why would you sandbox this?

Reply:
installing the service file, enabling it, pulling in fresh deps, etc.

docker makes this easy

also why not sandbox if you can

Reply:
i don't understand whats hard about this. docker probably costs more time to set up than just moving the service file, reloading the daemon, enabling it, and installing with pipenv

yes, i hate docker. thank @aeshthetic for that

Reply:
there's nothing stopping you from ignoring the Dockerfile and deploying it your own way locally. meanwhile a Dockerfile could automatically do all of the deployment steps for me. all I'd have to do is rebuild the container and bring it up again.

I'd like to learn how kubernetes works eventually as well, as that's probably The Right Way™.

maybe you hate docker but even if it were added, it would probably not affect you (also, it seems silly to hate a useful technology)

and I think this would be better discussed in #projects than in this GitHub thread because perhaps other people have relevant opinions also